bin/linter.dart

```diff
@@ -141,21 +144,27 @@ void runLinter(List<String> args, LinterOptions initialLintOptions) {
   }
 
   var stats = options['stats'];
-  if (stats == true) {
+  var benchmark = options['benchmark'];
+  if (stats == true || benchmark == true) {
```

Comment:
Yes!  For some reason I thought I convinced myself that undefined would return `null` but yeah, much better. 👍